pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -628,9 +628,9 @@ message Image {
     // ID of the image.
     optional string id = 1;
     // Other names by which this image is known.
-    repeated string repo_tags = 2;
+    repeated string names = 2;
```

Comment:
still think repo_tags is more clear, it clearly implies the format should be `image_name:tag`.